.Rbuildignore

```diff
@@ -7,6 +7,7 @@
 ^LICENSE
 ^_pkgdown\.yml$
 ^README.*$
+^README.Rmd
```

Comment:
We have no Readme.rmd, just Readme.md. And `.*$` should capture all file extensions.